packages/core/src/components/FAB.tsx

```diff
@@ -110,26 +110,24 @@ const styles = StyleSheet.create({
 
 export default withTheme(FAB);
 
-export const SEED_DATA = [
-  {
-    name: "FAB",
-    tag: "FAB",
-    category: COMPONENT_TYPES.button,
-    description: "A round, mini FAB",
-    layout: {},
-    props: {
-      onPress: createActionProp(),
-      icon: createIconProp({
-        defaultValue: null,
-      }),
-      iconColor: createColorProp({
-        label: "Icon color",
-        defaultValue: null,
-      }),
-      bgColor: createColorProp({
-        label: "Bg color",
-        defaultValue: null,
-      }),
-    },
+export const SEED_DATA = {
+  name: "FAB",
+  tag: "FAB",
+  category: COMPONENT_TYPES.button,
+  description: "A round, mini FAB",
+  layout: {},
+  props: {
+    onPress: createActionProp(),
+    icon: createIconProp({
```

Comment:
I think the problem is that in the above code, the code is looking for a prop called `iconName`, but according to the seed data the prop is (incorrectly) called `icon`

```suggestion
    iconName: createIconProp({
```

Reply:
fixed